fix: Remove `Change to Default` button suggestion in login page (#36062)
RocketChat/Rocket.Chat: .changeset/breezy-ravens-design.md

---
"@rocket.chat/meteor": patch
"@rocket.chat/web-ui-registration": patch
---

> Removes a `Change to Default` button that change the language in login page

Reviewer: ```suggestion
Removes the `Change to Default` language button from the login page
```
